### DISCUSSION TITLE
How to install spaczz

### DISCUSSION THREAD

**Discussion by 1ZvEzD1K1** [explosion/spaCy#11898] [Help: Installation]
How install spaczz???

i try pip install scpaczz but... 

 src/cpp_process.cpp:253:12: fatal error: 'longintrepr.h' file not found
        #include "longintrepr.h"
                 ^~~~~~~~~~~~~~~
      1 error generated.
      error: command '/usr/bin/clang' failed with exit code 1
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for rapidfuzz
Failed to build rapidfuzz
ERROR: Could not build wheels for rapidfuzz, which is required to install pyproject.toml-based projects

**Comment by polm** [comment]
spaczz is a third-party library, so it's probably better to ask at [their repo](https://github.com/gandersen101/spaczz). If you look at the README it looks like development has recently resumed after not being active for a while.
